### DISCUSSION TITLE
bedrock Minecraft

### DISCUSSION THREAD

**Discussion by aelsharawi** [AdguardTeam/AdGuardHome#6899] [Q&A]
hello

https://www.youtube.com/watch?v=AW5X7-qnvLk
/
anyway to achieve this with adguard>?

**Comment by joshqou** [comment]
If you know the domains that BedrockConnect is overriding, then you could add the BedrockConnect server [as a resolver for those domains](https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#upstreams-for-domains). This would still the machine to run BedrockConnect, however you would no longer need to manually change the DNS server on your Xbox/PS/Switch/etc
